cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/query/IndexManagerImpl.java

```diff
@@ -1,38 +1,41 @@
-//
-//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
-//  except in compliance with the License. You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-//  Unless required by applicable law or agreed to in writing, software distributed under the
-//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
-//  either express or implied. See the License for the specific language governing permissions
-//  and limitations under the License.
-
-//
-// The metadata for an index is represented in the database table as follows:
-//
-//   index_name  |  index_type  |  field_name  |  last_sequence
-//   -----------------------------------------------------------
-//     name      |  json        |   _id        |     0
-//     name      |  json        |   _rev       |     0
-//     name      |  json        |   firstName  |     0
-//     name      |  json        |   lastName   |     0
-//     age       |  json        |   age        |     0
-//
-// The index itself is a single table, with a column for docId and each of the indexed fields:
-//
-//      _id      |   _rev      |  firstName   |  lastName
-//   --------------------------------------------------------
-//     miker     |  1-blah     |  Mike        |  Rhodes
-//     johna     |  3-blob     |  John        |  Appleseed
-//     joeb      |  2-blip     |  Joe         |  Bloggs
-//
-// There is a single SQLite index created on all columns of this table.
-//
-// N.b.: _id and _rev are automatically added to all indexes to allow them to be used to
-// project DocumentRevisions without the need to load a document from the datastore.
+/*
+ * Copyright © 2014, 2016 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+/**
+ * The metadata for an index is represented in the database table as follows:
+ *
+ * index_name|index_type|field_name|last_sequence
+ * -----------------------------------------------------------
+ * name|json|_id|0
+ * name|json|_rev|0
+ * name|json|firstName|0
+ * name|json|lastName|0
+ * age|json|age|0
+ *
+ * The index itself is a single table,with a column for docId and each of the indexed fields:
+ *
+ * _id|_rev|firstName|lastName
+ * --------------------------------------------------------
+ * miker|1-blah|Mike|Rhodes
+ * johna|3-blob|John|Appleseed
+ * joeb|2-blip|Joe|Bloggs
```

Comment:
Keep table spacing and format like https://github.com/cloudant/sync-android/pull/379/files#diff-b7df9e23ebfd740a214a3db0f44ee84eL25 ?